vaadin-bom/pom.xml

```diff
@@ -18,7 +18,7 @@
     <flow.version>1.0.0.alpha18</flow.version>
     <testbench.version>6.0.0.alpha4</testbench.version>
     <board.version>1.1.0</board.version>
```

Comment:
board version seems to be obsolete

Reply:
yes, I will remove it or change it before beta